templates/nextjs/test/contract.test.ts

```diff
@@ -0,0 +1,59 @@
+import { launchTestNode } from 'fuels/test-utils';
+
+import { describe, test, expect } from 'vitest';
+
+/**
+ * Imports for the contract factory and bytecode, so that we can use them in the test.
+ *
+ * Can't find these imports? Make sure you've run `fuels build` to generate these with typegen.
+ */
+import { TestContractAbi__factory } from '../src/sway-api';
+import bytecode from '../src/sway-api/contracts/TestContractAbi.hex';
+
+/**
+ * @group node
+ *
+ * Tests for the contract program type within the TS SDK. Here we will test the deployment of
+ * our contract, and the result of call it's functions.
+ */
+describe('Contract', () => {
+  test('Deploy and Call', async () => {
+    // First, we'll launch a test node, passing the contract factory and bytecode. This will deploy the contract
+    // to our test node so we can test against it.
+    using launched = await launchTestNode({
```

Comment:
```suggestion
    The test node will be killed automatically once the `launched` variable goes out of scope,
    because we are instantiating it with the `using` keyword.
    using launched = await launchTestNode({
```

Reply:
Have you misclicked the lines for which you'd like to apply this suggestion?

It looks like it's breaking the code.

Reply:
@arboleya it sure is breaking it, I forgot to add the `//` comment slashes. I updated the suggestion.

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2755/commits/16635c3d958983c5da92fb75265398990c396458